test/plugin.test.js

```diff
@@ -631,3 +631,239 @@ QUnit.test('disposing the player also disposes the playlist menu', function(asse
   assert.strictEqual(this.fixture.querySelectorAll('.vjs-playlist').length, 1, 'only the unused playlist container is left');
   assert.strictEqual(this.player.playlistMenu, null, 'the playlistMenu property is null');
 });
+
+QUnit.module('videojs-playlist-ui: add/remove', {beforeEach: setup, afterEach: teardown});
```

Comment:
Definitely not blocking, but another test for both add/remove could be testing an index that is outside of the playlist length.